### PR TITLE
Update docs to use jsonforms v2.3.0

### PR DESCRIPTION
**Pull request by AlexandraBuzila**
- fix renderers
- fix small typos and rephrasals
- move `Multiple Forms` page to Tutorials 
- add Prettier config

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (edgarmueller) [inline .vscode/extensions.json:1]: Can we extract updating the `.vscode` settings into an own commit?

File: src/pages/docs/custom-renderers.mdx [hunk 2]
```diff
@@ -84,29 +84,34 @@ In this case, the props are `data`, which is the actual data bit to be rendered,
 a dot-separated string, which is necessary to propagate an update back to the store.
 
 For the `onClick` prop we pass the `handleChange` handler, which we retrieve
-via another helper function provided by JSON Forms called `withJsonFormsControlProps`.
+via another helper function provided by JSON Forms called `mapDispatchToControlProps`.
```
Comment (edgarmueller) [inline src/pages/docs/custom-renderers.mdx]: `withJsonFormsControlProps` is correct.

File: src/pages/docs/custom-renderers.mdx [hunk 3]
```diff
@@ -84,29 +84,34 @@ In this case, the props are `data`, which is the actual data bit to be rendered,
...
+const RatingControl = ({ data, handleChange, path }: RatingControlProps) => (
   <Rating
     value={data}
-    onClick={ev => handleChange(path, Number(ev.value))}
+    onClick={(ev: any) => handleChange(path, Number(ev.value))}
   />
 );
 
-export default withJsonFormsControlProps(RatingControl);
+export default connect(
```
Comment (edgarmueller) [inline src/pages/docs/custom-renderers.mdx]: Same here, keep `withJsonFormsControlProps`

File: src/pages/docs/layouts.mdx [hunk 4]
```diff
@@ -49,12 +55,13 @@ horizontal fashion. Each child occupies the same amount of space, i.e. for n chi
 }
 ```
 
-<DemoButton to='/examples/layouts#horizontallayout' label="Demo"/>
+<DemoButton to='/examples/layouts#horizontal-layout' label='Demo' />
```
Comment (edgarmueller) [inline src/pages/docs/layouts.mdx]: AFAIK the anchors are generated, so is this working with the additional hyphen?

File: src/pages/docs/multiple-forms.mdx [hunk 5]
```diff
@@ -1,29 +1,34 @@
...
-There might be use cases where you have forms that do not have anything in common. In such cases
-you can explicitly pass  `schema` and `uischema` props to the `JsonForms` component
+## Independent forms
+
+There might be use cases where you have forms that do not have anything in common, so your forms are independent, although they share the same store. In such cases
+you can explicitly pass `schema` and `uischema` props to the `JsonFormsDispatch` component
 (if you don't specify these, they will be fetched from the store).
 
-Additionally, the `JsonForms` component also takes a `path` prop which specifies
+Additionally, the `JsonFormsDispatch` component also takes a `path` prop which specifies
```
Comment (edgarmueller) [inline src/pages/docs/multiple-forms.mdx]: `JsonForms` is the right one..

File: src/pages/docs/multiple-forms.mdx [hunk 6]
```diff
@@ -35,11 +40,11 @@ The code for the above example looks as follows:
 
 ```
 <div>
-  <JsonForms schema={personSchema}
+  <JsonFormsDispatch schema={personSchema}
```
Comment (edgarmueller) [inline src/pages/docs/multiple-forms.mdx]: Same here, `JsonFormsDispatch` needs to be used when using Redux.

**Review by edgarmueller — CHANGES_REQUESTED** [review]
Thanks for updating the docs! I think a couple of things got mixed up, could you have a look again?

File: .vscode/extensions.json (hunk 1 above)
Comment (AlexandraBuzila) [reply .vscode/extensions.json:1]: Done, added a new commit that adds those configuration files.

File: src/pages/docs/custom-renderers.mdx (hunk 2 above)
Comment (AlexandraBuzila) [reply src/pages/docs/custom-renderers.mdx]: Reverted the changes.

File: src/pages/docs/custom-renderers.mdx (hunk 3 above)
Comment (AlexandraBuzila) [reply src/pages/docs/custom-renderers.mdx]: Reverted the changes.

File: src/pages/docs/layouts.mdx (hunk 4 above)
Comment (AlexandraBuzila) [reply src/pages/docs/layouts.mdx]: It does not work with the hyphen and it does not work with it either. I searched and this is a bug in docz. I removed the hyphen.

File: src/pages/docs/multiple-forms.mdx (hunk 5 above)
Comment (AlexandraBuzila) [reply src/pages/docs/multiple-forms.mdx]: Done, reverted the change.

File: src/pages/docs/multiple-forms.mdx (hunk 6 above)
Comment (AlexandraBuzila) [reply src/pages/docs/multiple-forms.mdx]: Done, reverted the change.

File: src/pages/docs/multiple-forms.mdx (hunk 5 above)
Comment (edgarmueller) [reply src/pages/docs/multiple-forms.mdx]: I think this one is still outdated.

Comment (edgarmueller) [reply src/pages/docs/multiple-forms.mdx]: Actually, I think this whole paragraph is not true anymore, so we might get rid of it altogether.

Comment (AlexandraBuzila) [reply src/pages/docs/multiple-forms.mdx]: I removed the paragraph.

**Review by edgarmueller — APPROVED** [review]

**Comment by edgarmueller** [comment]
Thanks!

**Pull request merged**